DeepResearch-Backend/routers/chat_router.py

```
@@ -0,0 +1,132 @@
+        # AiChatDataStreamAsyncResponse will call streamer.async_stream(prompt, messages)
+        return AiChatDataStreamAsyncResponse(
+            streamer,
+            prompt=latest_user_message,
+            messages=messages_dict
+        )
+
+    except HTTPException:
+        # Re-raise validation errors
+        raise
+    except Exception as e:
+        logger.error(f"Unexpected error in chat endpoint: {str(e)}", exc_info=True)
```

Ruff (0.15.6)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)